### PR TITLE
fix(wiki-settings): include questy settings in details

### PR DESCRIPTION
**Pull request by m90**
Ticket https://phabricator.wikimedia.org/T354954

### CODE REVIEW

File: app/Wiki.php [hunk 1]
```diff
@@ -138,6 +138,8 @@ public function publicSettings()
             'wwWikibaseStringLengthMultilang',
             'wikibaseFedPropsEnable',
             'wikibaseManifestEquivEntities',
+            'wwUseQuestyCaptcha',
+            'wwCaptchaQuestions',
```
Comment (m90) [inline app/Wiki.php:142]: I wonder if we should make this a blocklist instead? Else we'll be bitten by this again I surmise.

Comment (tarrow) [reply app/Wiki.php:142]: Yeah, I guess things like wgSecret and the db credentials are things we would want to not expose to the manager but this list is fairly short.

Comment (m90) [reply app/Wiki.php:142]: Today I learned there are DB credentials in `wikiSettings`. Then maybe let's keep it like this.

**Review by tarrow — APPROVED** [review]
cool; let's go ahead with this. I also double checked if we might be leaking something via PublicWikiResource or something but it appears not.

**Pull request merged**